src/Microsoft.Identity.Client/ClientApplicationBase.WithBuilders.cs

```diff
@@ -55,14 +55,14 @@ internal ClientApplicationBase(ApplicationConfiguration config)
                 UserTokenCacheInternal = new TokenCache(ServiceBundle);
             }
 
-            CreateRequestContext().Logger.Info(
-                string.Format(
-                    CultureInfo.InvariantCulture,
-                    "MSAL {0} with assembly version '{1}', file version '{2}' and informational version '{3}' is running...",
-                    ServiceBundle.PlatformProxy.GetProductName(),
-                    MsalIdHelper.GetMsalVersion(),
-                    AssemblyUtils.GetAssemblyFileVersionAttribute(),
-                    AssemblyUtils.GetAssemblyInformationalVersion()));
+            //CreateRequestContext().Logger.Info(
```

Comment:
remove or modify this?

Reply:
Yeah, I need to figure this one out.  ON the .NET desktop app, we get a crash since we're doing this in the constructor and the constructor is called from the mainform constructor and the callback sends a message dispatch to the main window to show the log output.  so we shouldn't be doing work from our constructor and causing this problem.  however, there's not really a good follow up place to put this info which is probably useful for logging